common/schema-builder/src/main/java/io/smallrye/graphql/schema/helper/BeanValidationDirectivesHelper.java

```diff
@@ -91,6 +101,32 @@ public List<DirectiveInstance> transformBeanValidationConstraintsToDirectives(An
                 directive.setValue("min", getLongValue(annotations, annotationName, "value"));
                 result.add(directive);
             }
+            if (annotationName.equals(VALIDATION_ANNOTATION_DECIMAL_MAX)) {
+                DirectiveInstance directive = new DirectiveInstance();
+                directive.setType(CONSTRAINT_DIRECTIVE_TYPE);
+                String value = getStringValue(annotations, annotationName, "value");
+                try {
+                    directive.setValue("maxFloat", new BigDecimal(value).doubleValue());
+                    result.add(directive);
+                } catch (NumberFormatException nfe) {
+                    LOGGER.warn(
+                            "Not generating a bean validation directive for " + annotations.getAnnotationValue(annotationName) +
+                                    " because the value can't be parsed as a BigDecimal");
+                }
+            }
+            if (annotationName.equals(VALIDATION_ANNOTATION_DECIMAL_MIN)) {
+                DirectiveInstance directive = new DirectiveInstance();
+                directive.setType(CONSTRAINT_DIRECTIVE_TYPE);
+                String value = getStringValue(annotations, annotationName, "value");
+                try {
+                    directive.setValue("minFloat", new BigDecimal(value).doubleValue());
```

Comment:
I would use BigDecimal directly but then it blows up because Quarkus bytecode recording doesn't support that type, so I have to convert to double...

Reply:
But that will blow up if it's larger than a double, won't it?

Reply:
I double checked, it won't throw an exception but lose precision.

Reply:
... which is okay, I guess. Maybe add a comment?